coretex/_task/base_callback.py

```diff
@@ -50,7 +50,10 @@ def onStart(self) -> None:
         # Call "kill -30 task_run_process_id" to dump current stack trace of the TaskRun into the file
         # 30 == signal.SIGUSR1
         # Only works on *nix systems
-        faultHandlerPath = folder_manager.logs / f"task_run_stacktrace_{self._taskRun.id}_{datetime.now().strftime(DATE_FORMAT)}.log"
+        taskRunLogs = folder_manager.logs / "runs" / str(self._taskRun.id)
+        taskRunLogs.mkdir(parents = True, exist_ok = True)
```

Comment:
Let's extract `folder_manager.logs / "runs"` to `folder_manager` module so we can access it like this `folder_manager.runsLogDirectory`